README.md

```diff
@@ -43,7 +43,7 @@ Get started
 ---
 
 1. Download a release and unzip or clone into a directory
-2. Enter the directory and type `npm install && bower install`, which will install all the dependencies
+2. Enter the directory and type `bin/setup`, which will install all the dependencies and create config file
```

Comment:
Please add more description for prerequisite of npm and need a bash to run the setup script.